cisco_aci/datadog_checks/cisco_aci/models.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import six
+
+if six.PY3:
+    from typing import Optional
+
+    from pydantic import BaseModel, Field, computed_field
+
+    class NodeAttributes(BaseModel):
+        address: Optional[str] = None
+        ad_st: Optional[str] = Field(default=None, alias="adSt")
+        role: Optional[str] = None
+        dn: Optional[str] = None
+        model: Optional[str] = None
+        version: Optional[str] = None
+        serial: Optional[str] = None
+        vendor: Optional[str] = Field(default='cisco_aci')
+        namespace: Optional[str] = Field(default='default')
+
+    class Node(BaseModel):
+        attributes: NodeAttributes
+
+    class EthAttributes(BaseModel):
+        admin_st: Optional[str] = Field(default=None, alias="adminSt")
+        id: Optional[str] = None
+        name: Optional[str] = None
+        desc: Optional[str] = None
+        router_mac: Optional[str] = Field(default=None, alias="routerMac")
+
+    class Eth(BaseModel):
+        attributes: EthAttributes
+
+    class DeviceMetadata(BaseModel):
```

Comment:
We're missing `description`, `sys_object_id`, `location`, `profile`, `profile_version`, `subnet`, `product_name`, `os_name`, `os_version`, `os_hostname`, `integration` and `device_type`.

Is there anything we can add? At least `integration` ?